src/components/SettlementButton.tsx

```diff
@@ -227,12 +213,12 @@ function SettlementButton({
             anchorAlignment={kycWallAnchorAlignment}
             shouldShowPersonalBankAccountOption={shouldShowPersonalBankAccountOption}
         >
-            {(triggerKYCFlow, buttonRef) => (
+            {(triggerKYCFlow: TriggerKYCFlow, buttonRef: ForwardedRef<HTMLElement>) => (
                 <ButtonWithDropdownMenu
-                    buttonRef={buttonRef}
+                    buttonRef={buttonRef as RefObject<View>}
```

Comment:
Please console.log the type of buttonRef here, I suspect the type is wrong in either KYCWall or ButtonWithDropdownMenu

Reply:
Shouldn't it be `HTMLElement | View` in `KYCWall` then?

Reply:
Please use `src/types/utils/viewRef.ts` util here and change types in both files to `View | HTMLDivElement` if possible